app/controllers/api/reviews_controller.rb

```diff
@@ -0,0 +1,46 @@
+class Api::ReviewsController < ApiController
+
+  def create
+    user = User.find_by_name(params['user_name'])
+
+    unless user
+        user = User.create({ :name => params['user_name']})
```

Comment:
```ruby
User.create(name: params[:user_name])
```

Also, check out the "bang" functions - `create!`, `save!`, and check out how `rescue_from(SomeException)` works.